cudax/include/cuda/experimental/__execution/policy.cuh

```diff
@@ -44,48 +44,48 @@ struct any_execution_policy
 
   _CCCL_HIDE_FROM_ABI any_execution_policy() = default;
 
-  template <__execution_policy _Policy>
-  _CCCL_HOST_API constexpr any_execution_policy(::cuda::std::execution::__policy<_Policy>) noexcept
-      : value(_Policy)
+  template <uint32_t _Policy>
```

Comment:
suggestion (@gonidelis): we use `uint8` in some places and `int32` in other places, maybe consider having a type alias somewhere?